users/models.py

```diff
@@ -274,3 +275,41 @@ def create_last_seen_for_user(sender, instance, created, raw, **kwargs):
 
     if created:
         LastSeen.objects.create(user=instance, at=timezone.now())
+
+
+class InvitationToken(models.Model):
+    """
+    An invitation token to create a user account.
+    """
+
+    user = models.ForeignKey(
```

Comment:
Does an invitation needs a way to track whether it was consumed/user or not? (besides expiration)